bodhi/server/util.py

```diff
@@ -343,6 +343,20 @@ def status2html(context, status):
     return "<span class='label label-%s'>%s</span>" % (cls, status)
 
 
+def ci_status2html(context, status):
+    cls = {
+        'Ignored': 'success',
+        'Running': 'warning',
+        'Passed': 'success',
+        'Failed': 'danger',
+        'Queued': 'info',
+        'None': 'primary',
```

Comment:
We can use the Enums here too so we don't have to do string comparison - the return statement below should automatically convert the Enum to human readable form, I think.